packages/phone-number-privacy/combiner/src/pnp/endpoints/quota/action.ts

```diff
@@ -0,0 +1,34 @@
+import { ErrorMessage, PnPQuotaRequest } from '@celo/phone-number-privacy-common'
+import { CombineAction } from '../../../common/combine'
+import { IO } from '../../../common/io'
+import { Session } from '../../../common/session'
+import { OdisConfig } from '../../../config'
+import { CombinerThresholdStateService } from '../../services/thresholdState'
+
+export class PnPQuotaAction extends CombineAction<PnPQuotaRequest> {
+  constructor(
+    readonly config: OdisConfig,
+    readonly thresholdStateService: CombinerThresholdStateService<PnPQuotaRequest>,
+    readonly io: IO<PnPQuotaRequest>
```

Comment:
This is probably something I could've made clearer with naming. The IO class is abstract, and we use its abstract functions in this file. That means we need to implement an IO class for this endpoint that extends the abstract class  and use that here. I can help you with that in a quick 1:1!